libs/langgraph/langgraph/errors.py

```diff
@@ -22,13 +22,22 @@ class GraphRecursionError(RecursionError):
         )
     """
 
-    pass
+    def __init__(
+        self,
+        message: str = "Recursion limit reached.\n\nTroubleshooting URL: https://python.langchain.com/docs/troubleshooting/errors/GRAPH_RECURSION_LIMIT/",
```

Comment:
I don't think this will actually get used as-written. I think we need to either append to the message passed in, or log a message

Reply:
why? it works for me

Reply:
I think all instances of `raise GraphRecursionError` have a message set, so this wouldn't populate right?

https://github.com/langchain-ai/langgraph/blob/main/libs/langgraph/langgraph/pregel/__init__.py#L1310

https://github.com/langchain-ai/langgraph/blob/main/libs/langgraph/langgraph/pregel/__init__.py#L1516

Reply:
Yeah looks like a mistake